synapse/config/sso.py

```diff
@@ -110,6 +114,12 @@ def generate_config_section(self, **kwargs):
             #client_whitelist:
             #  - https://riot.im/develop
             #  - https://my.custom.client/
+            #
+            # Keep a user's profile fields in sync with the values from the SSO ticket
+            # (currently display_name). The update is performed on every SSO login.
```

Comment:
In line with what @clokep [noted above](https://github.com/matrix-org/synapse/pull/10108#issuecomment-853261726), we should inform the sysadmin that setting this option to `true` will overwrite profile information of users from the SSO ticket, regardless of whether they've chosen to opt out of importing that information when first logging in or not.

```suggestion
            # Keep a user's profile fields in sync with the values from the SSO ticket
            # (currently display_name). The update is performed on every SSO login.
            #
            # Note that enabling this option will override user profile information from
            # the SSO ticket, regardless of whether users have opted-out of syncing
            # that information when first signing in. Defaults to false.
```